dist/svg.js

```diff
@@ -1694,20 +1704,53 @@ SVG.FX = SVG.invent({
 
       // convert current time to position
       if(!ignoreTime) this.pos = this.timeToPos(+new Date)
+      else {
+        // If the method at was called with a position above 1, we set it back to 1
+        if(this.pos > 1) this.pos = 1
+        if(this.situation.reversed) this.pos = 1 - this.pos
+      }
 
-      if(this.pos >= 1 && (this.situation.loop === true || (typeof this.situation.loop == 'number' && --this.situation.loop))){
+      if(this.pos < 0) this.pos = 0
+      if(this.pos >= 1){
```

Comment:
Well if we allow a call like `at(1.3)` to start the next loop that would definitely simplify thing. But, if we allow that, we would also have to allow, when the animation is reversed, a call like `at(-0.3)` to also start the next loop. Having a negative position looked weird to me, so that is why I limited the values at can receive to between 0 and 1.